lib/flatfile/config/QubitRelation-actor.yml

```diff
@@ -0,0 +1,9 @@
+# Columns that should be included in the actor relation iport
```

Comment:
Nitpick: Typo "iport" should be "import" :)

Reply:
D'oh! I'll fix. Just realized I've got to update the columns in this file too.

Reply:
I think renaming "sourceAuthorizedFormOfName" to "objectAuthorizedFormOfName" and "targetAuthorizedFormOfName" to "subjectAuthorizedFormOfName" might be clearer too...